lib/core/src/control/io/mouse2.rs

```diff
@@ -109,3 +111,30 @@ define_bindings! {
     MouseEvent::mousemove  => on_move  (OnMove),
     WheelEvent::mousewheel => on_wheel (OnWheel),
 }
+
+/// A handles of callbacks emitting events on bound FRP graph. See `CallbackHandle`.
+#[derive(Debug)]
+pub struct MouseFrpCallbackHandles {
+    on_move : CallbackHandle,
+    on_down : CallbackHandle,
+    on_up   : CallbackHandle,
+}
+
+/// Bind FRP graph to MouseManager.
+pub fn bind_frp_to_mouse(frp:&enso_frp::Mouse, mouse_manager:&MouseManager)
+-> MouseFrpCallbackHandles {
+    let height        = web::window().inner_height().unwrap().as_f64().unwrap() as i32;
```

Comment:
1. Hardcoded scene size (you hsohuld use the scene size which we keep in a shared object here)
2. Try using JS things as rarely as you can, so we track progress of that easily to minimize the possibility of reflow hapenning :)